docs/docs/releases/0.2.0.md

```diff
@@ -1,302 +1,550 @@
+[#3795](https://github.com/mathesar-foundation/mathesar/pull/3795 "Add privilege information fields")
+[#3810](https://github.com/mathesar-foundation/mathesar/pull/3810 "Database permissions modal - 'Share' tab")
+[#3833](https://github.com/mathesar-foundation/mathesar/pull/3833 "Implement `transfer_ownership` for schemas and tables")
+[#3836](https://github.com/mathesar-foundation/mathesar/pull/3836 "Implement Permissions modal -> 'Share' tab for Schemas")
+[#3838](https://github.com/mathesar-foundation/mathesar/pull/3838 "Implement permissions modal for schemas and tables")
+[#3840](https://github.com/mathesar-foundation/mathesar/pull/3840 "Allow setting owner at Creation")
+[#3858](https://github.com/mathesar-foundation/mathesar/pull/3858 "Implement `roles.delete`, `databases.drop` & `databases.configured.disconnect` endpoints")
+[#3859](https://github.com/mathesar-foundation/mathesar/pull/3859 "Implement 'Transfer ownership' tabs")
+[#3861](https://github.com/mathesar-foundation/mathesar/pull/3861 "permission checks in all pages, Disconnect database option")
+[#3869](https://github.com/mathesar-foundation/mathesar/pull/3869 "Bugfix `get_current_role` when only one role exists")
+[#3881](https://github.com/mathesar-foundation/mathesar/pull/3881 "Grant appropriate permissions for `msar`, `__msar` and `mathesar_types` to `PUBLIC`")
+[#3895](https://github.com/mathesar-foundation/mathesar/pull/3895 "Filter databases for admin and standard users")
+[#3922](https://github.com/mathesar-foundation/mathesar/pull/3922 "Fix logic in `get_current_role`")
+[#3931](https://github.com/mathesar-foundation/mathesar/pull/3931 "Fix quoting for role grant/revoke expressions")
+[#3958](https://github.com/mathesar-foundation/mathesar/pull/3958 "Bugfix msar.set_members")
+[#3966](https://github.com/mathesar-foundation/mathesar/pull/3966 "Fix missing links in cell context menu")
+[#3982](https://github.com/mathesar-foundation/mathesar/pull/3982 "Fix some bugs from milestone testing-2")
+[#4011](https://github.com/mathesar-foundation/mathesar/pull/4011 "Return correct permission denied exception from SQL functions")
+[#4024](https://github.com/mathesar-foundation/mathesar/pull/4024 "Allow only superusers to disconnect configured DBs")
+[#4082](https://github.com/mathesar-foundation/mathesar/pull/4082 "Added Table Permission Modal in Table Card Menu")
+[#4104](https://github.com/mathesar-foundation/mathesar/pull/4104 "Fix confusing role field on user edit form")
+[#4132](https://github.com/mathesar-foundation/mathesar/pull/4132 "Handle errors better, Show a clearer error when user is not a collaborator on a database")
+[#4075](https://github.com/mathesar-foundation/mathesar/pull/4075 "Generated Identity for example datasets")
+
+### Substantial performance improvements {:#performance}
+
+Mathesar is now much faster! Most pages load about 5x faster, and large imports run about 50x faster.
```

Comment:
You could add a bit about the fact that "real world installations" (i.e., ones with more latency between the DB and service) will see even greater improvements.

Reply:
I'm not going to bother with this for now because I'm short on time. If I do spend more time on this, I'd like to actually put some bar charts in there. I might do that after the release depending on priorities.